src/Raven.Client/ServerWide/Operations/SetDatabaseDynamicDistributionOperation.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class SetDatabaseDynamicDistributionOperation : IServerOperation<bool>
```

Comment:
`IServerOperation<bool>` needs to be changed to `IServerOperation`
The endpoint does not return anything

Reply:
I missed that interface, was looking for it, was below the other one